lib/reducers/call-taker.js

```diff
@@ -4,15 +4,18 @@ import moment from 'moment'
 import {constructNewCall} from '../util/call-taker'
 import {FETCH_STATUS} from '../util/constants'
 
-function createCallTakerReducer () {
+function createCallTakerReducer (config) {
+  const calltakerConfig = config.modules.find(m => m.id === 'call')
   const initialState = {
     activeCall: null,
     callHistory: {
       calls: {
         status: FETCH_STATUS.UNFETCHED,
         data: []
       },
-      visible: false
+      visible: calltakerConfig &&
+        calltakerConfig.options &&
+        calltakerConfig.options.showCallHistoryOnLoad
```

Comment:
There's some new JS syntax called [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) that makes this a lot cleaner:
```suggestion
      visible: calltakerConfig?.options?.showCallHistoryOnLoad
```